bin/clark

```diff
@@ -1,5 +1,37 @@
 #!/usr/bin/env node
+
 'use strict';
 
-// eslint-disable-next-line import/no-unassigned-import
-require('../lib/cli');
+const fs = require('fs');
+const path = require('path');
+
+const findRoot = require('find-root');
+
+const {makeDebug} = require('../lib/lib/debug');
+
+const debug = makeDebug(__filename);
+
+try {
+  debug('looking for locally installed clark');
+  const root = findRoot(process.cwd(), dir => {
+    const clarkPath = path.resolve(dir, 'node_modules', '.bin', 'clark');
+    // eslint-disable-next-line no-sync
+    return fs.existsSync(clarkPath);
+  });
+
+  debug('found locally installed clark');
+  const localPath = path.resolve(root, 'node_modules', '.bin', 'clark');
+  debug(`loading clark at ${localPath}`);
+  require(localPath);
```

Comment:
Should this instead require the cli file? Otherwise I think it will keep looking upward?

Reply:
ah, yea. i was testing locally against the version installed in `server`, so i didn't run into that. fixing it.